tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -353,6 +353,31 @@ class SomeClass extends \Foo\Bar\A implements \Foo\Bar\Izumi, A, \A\B, \Foo\Bar\
             '<?php use A\B; class Foo { use \A\C; use \D; use \A\B { \A\B::bar as baz; } };',
         ];
 
+        yield 'typed property in function' => [
+            '<?php use A\B; function foo(B $v) {}',
+            '<?php use A\B; function foo(\A\B $v) {}',
+        ];
```

Comment:
It was fixed already and is covered with tests, no need to add another case.

Reply:
what test case name do you mean to remove and where it is already tested?

Reply:
removed